http/headers/Permissions-Policy.json

```diff
@@ -186,22 +161,48 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Permissions-Policy/autoplay",
             "spec_url": "https://html.spec.whatwg.org/multipage/infrastructure.html#autoplay-feature",
             "support": {
-              "chrome": [
-                {
-                  "version_added": "88"
-                },
-                {
-                  "alternative_name": "Feature-Policy",
-                  "version_added": "64"
-                }
-              ],
+              "chrome": {
+                "version_added": "64"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "battery": {
```

Comment:
Mind adding `battery` in a separate PR?

Reply:
To avoid merge conflicts, I feel it is best to keep it all in the same PR -- but if still desired, I can split it out!

Reply:
I have removed it in 521b2de66 and we can simply revert that in a separate PR after merging this, which shouldn't cause a merge conflict.